homeassistant/components/vicare/strings.json

```diff
@@ -242,6 +242,44 @@
       "volumetric_flow": {
         "name": "Volumetric flow"
       },
+      "ess_state_of_charge": {
+        "name": "Battery"
+      },
+      "ess_power_current": {
+        "name": "Battery power"
+      },
+      "ess_state": {
+        "name": "Battery state",
+        "state": {
+          "charge": "Charge",
+          "discharge": "Discharge",
+          "standby": "Standby"
+        }
+      },
+      "pcc_current_power_exchange": {
+        "name": "PCC power exchange"
+      },
+      "pcc_energy_consumption": {
+        "name": "PCC energy consumption"
+      },
+      "pcc_energy_feed_in": {
+        "name": "PCC energy feed-in"
+      },
+      "photovoltaic_power_production_current": {
+        "name": "Solar power"
+      },
+      "photovoltaic_energy_production_today": {
+        "name": "Solar energy production today"
+      },
+      "photovoltaic_status": {
+        "name": "Solar state",
+        "state": {
+          "nothing": "Nothing",
+          "ready": "Ready",
+          "production": "Production",
```

Comment:
What does Nothing and Production mean? I am trying to put it in a question like: "What's the solar state?" - "Production" or "Nothing". Doesn't really fit imo.

Maybe "Idle" and "In production"/"Producing"/"Active"?

Reply:
@CHTHSCH @DenisFFM can you shed some light on this as you own such devices?

Reply:
In the ViCare app, I have seen states like
Production
Standby
Ready

For everything else I may refer to the Viessmann documentation I shared

Reply:
What could be the difference between ready and standby?

Reply:
I would say ready = Standby. Perhaps you change that in the integration.

Here is the link to the user manual [https://static.viessmann.com/resources/technical_documents/CH/de/VBA/6176080VBA00002_1.pdf?#pagemode=bookmarks&zoom=page-fit&view=Fit](url)

I am verry sorry, but I am and the end of my knowledge to help further.

Reply:
As an end user, I am ok with how @CFenner has implemented the states

ready = Standby => the system is waiting for the sun to shine
Production => the system is generating electricity
unknown or nothing => none